### PR TITLE
Sparkline error bar test

### PR DESCRIPTION
**Pull request by mj719**
error bar test

### CODE REVIEW

**Comment by elong0527** [comment]
Please double check the testing code, the `test-coverage` workflow shall pass before we merge. 

https://github.com/elong0527/forestly/pull/34/checks?check_run_id=3070542985

File: tests/testthat/test-sparkline_errorbar.R [hunk 1]
```diff
@@ -0,0 +1,29 @@
+library(testthat)
+
```
Comment (wangben718) [inline tests/testthat/test-sparkline_errorbar.R:5]: library(plotly)
library(ggplot2)

File: tests/testthat/test-sparkline_errorbar.R [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                                upper = c(3, 5, 5),
+                                color  = c("red", "blue", "gold")))
+  expect_error(sparkline_errorbar(lower=c(-1, -2, -1),
+                                upper = c(3, 5, 5),
+                                color  = c("red", "blue", "gold")))
+})
+
+test_that("sparkline_errorbar can generate error bars", {
+
+  expect_s3_class(tidy_observation(est = c(0,1,2),
```
Comment (wangben718) [inline tests/testthat/test-sparkline_errorbar.R]: replace `tidy_observation()` to `sparkline_errorbar()'

File: tests/testthat/test-sparkline_errorbar.R [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                                   lower = c(-1, -2, -1),
+                                   upper  = c(3, 5, 5),
+                                   color  = c("red", "blue", "gold")))
+})
```
Comment (wangben718) [inline tests/testthat/test-sparkline_errorbar.R:20]: plotly_snap <- function(p){
  p_js <- plotly::plotly_json(p, jsonedit = FALSE)
  p_lst <- jsonlite::fromJSON(p_js)

  gsub(names(p_lst$visdat), "0000", as.character(p_lst))

}

test_that("sparkline_errorbar",{
  est <- c(0,1,2)
  lower <- c(-1, -2, -1)
  upper <- c(3, 5, 5)
  color <- c("red", "blue", "gold")
  temp1<-sparkline_errorbar(est,lower,upper,color)

  expect_snapshot_output(plotly_snap(temp1))
})

**Review by wangben718 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**